RealmSwift/SyncSubscription.swift

```diff
@@ -192,15 +214,19 @@ import Realm.Private
 
     /**
      Synchronously performs any transactions (add/remove/update) to the subscription set within the block.
-     This will not wait for the server to acknowledge and see all the data associated with this collection of subscriptions,
-     and will return after committing the subscription transactions.
```

Comment:
Were the old docs just wrong here?

Reply:
This may be a little bit confusing to the user, because it does complete when the subscription is completed and data bootstrapped if `onComplete` is not nil.